src/GitVersionCore/Configuration/NamedConfigFileLocator.cs

```diff
@@ -35,8 +35,12 @@ private void WarnAboutAmbiguousConfigFileSelection(string workingDirectory, stri
             var workingConfigFile = GetConfigFilePath(workingDirectory, fileSystem);
             var projectRootConfigFile = GetConfigFilePath(projectRootDirectory, fileSystem);
 
+            if (Path.GetFullPath(workingConfigFile).Equals(Path.GetFullPath(projectRootConfigFile)))
```

Comment:
Will `GetFullPath` normalize the path so a `String.Equals` is sufficient here? I.e. are trailing slashes removed (or added) consistently, folder names adjusted to the physical casing (on case-insensitive file systems), etc.?

Reply:
it will normalize the paths (i.e. slashes and paths get normalized) however, casing does not.  
Trailing slashes aren't normalized, but these are filenames, so trailing slashes are not supported.  It would be worth making this case insensitive, however, so I'll post a commit for that.

Reply:
I've updated this to behave correctly for case-insensitive platforms.